src/hmem.c

```diff
@@ -349,3 +355,15 @@ bool ofi_hmem_is_ipc_enabled(enum fi_hmem_iface iface)
 {
 	return hmem_ops[iface].is_ipc_enabled();
 }
+
+int ofi_hmem_get_ipc_handle_size(enum fi_hmem_iface iface, size_t *size)
+{
+	int ret;
+	ret = hmem_ops[iface].get_ipc_handle_size(size);
+	if (ret)
+		FI_WARN(&core_prov, FI_LOG_CORE,
+		"Failed to get ipc handle size with hmem iface %s: %s\n",
+		fi_tostr(&iface, FI_TYPE_HMEM_IFACE),
+		fi_strerror(-ret));
+	return ret;
+}
```

Comment:
Have this function return size_t and always return the size.  The call should not fail.

Reply:
If we make it return `size_t`, how do we handle the case of `HAVE_CUDA=0` ? return 0?

Reply:
Yes, I would just return 0.  We should never get far enough to ever call this function in the no-op case.

Reply:
Thank you! Updated.